apps/etterlatte-saksbehandling-ui/client/src/components/behandling/revurderingsoversikt/NavnInput.tsx

```diff
@@ -0,0 +1,71 @@
+import { Navn } from '~shared/types/RevurderingInfo'
+import React, { useState } from 'react'
+import { TextField } from '@navikt/ds-react'
+import styled from 'styled-components'
+
+export const NavnInput = (props: { navn: Navn | undefined; update: (n: Navn) => void }) => {
+  const { navn, update } = props
+  const [fornavn, setFornavn] = useState(navn?.fornavn || '')
```

Comment:
Trenger vi disse? Kunne evt hatt default state navn som sendes inn, såvidt jeg ser er dette lokal state og. 
Da kunne vi kun hatt en update ala
´update({ ...navn,
fornavn}) `
For hver av textfieldsa i hvert fall. Men smak og behag, synes bare det virket litt rart å nesten lokalState på den måten her :P

Reply:
Ooh, liker tanken!